### PR TITLE
Issue 67: Fixed main.py imports

### PR DESCRIPTION
**Pull request by Alan-R**
Changed imports in main.py to import from our package instead.
Also added some information to setup.py to make a warning go away
and also give more package information.

@olsonse: please review.

### CODE REVIEW

**Comment by Alan-R** [comment]
This allows it to run on systems where the default installed version of Python is 2.7.

**Comment by Alan-R** [comment]
I just added the UNCHECKED fix to this branch as well.

**Comment by Alan-R** [comment]
Main.py gets installed in a system "bin" directory somewhere. When it's there, and not in /usr/lib/python*, it must import from packages, not as though it were part of a package. That's why I had to change the form of import.

**Comment by Alan-R** [comment]
Could be I did something wrong...

**Comment by Alan-R** [comment]
I have no idea why this failed for me. It's working now. Got no clue...

File: ctypesgen/printer_python/preamble/3_2.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def ReturnString(obj, func=None, arguments=None):
 # Non-primitive return values wrapped with UNCHECKED won't be
 # typechecked, and will be converted to c_void_p.
 def UNCHECKED(type):
-    if hasattr(type, "_type_") and isinstance(type._type_, bytes) and type._type_ != "P":
+    if hasattr(type, "_type_") and isinstance(type._type_, str) and type._type_ != "P":
```
Comment (olsonse) [inline ctypesgen/printer_python/preamble/3_2.py]: Are you going to cherry-pick this commit and merge this fix?  Perhaps with a test that demonstrates it working?

**Comment by Alan-R** [comment]
I agree I need to do both. I'll see what I can do after church today.

**Pull request closed without merging**